CodeGeneration/Sources/Utils/CodeGenerationFormat.swift

```diff
@@ -158,21 +158,30 @@ public class CodeGenerationFormat: BasicFormat {
     }
     formattedChildren = formattedChildren.map { child in
       var child = child
-      child.trailingTrivia = Trivia(pieces: child.trailingTrivia.drop(while: \.isSpaceOrTab))
+
+      let firstNonSpaceOrTabIndex =
+        child.trailingTrivia.firstIndex(where: { !$0.isSpaceOrTab }) ?? child.trailingTrivia.endIndex
```

Comment:
I think you can avoid the `??` if you push the optional unwrapping into the `if` condition.

Reply:
Like this?
```swift
if let firstNonSpaceOrTabIndex = child.trailingTrivia.firstIndex(where: { !$0.isSpaceOrTab }) {
  if child.trailingTrivia[firstNonSpaceOrTabIndex].isNewline {
    child.trailingTrivia = Trivia(pieces: child.trailingTrivia.suffix(from: firstNonSpaceOrTabIndex))
  }
} else {
  child.trailingTrivia = Trivia()
}
```

Reply:
Yes, that’s what I was thinking

Reply:
The change has been made 👍